src/components/DocumentationTopic/ContentTable.vue

```diff
@@ -15,18 +15,19 @@
     :title="title"
   >
     <div class="container">
-      <h2 class="title">{{title}}</h2>
+      <SectionTitle class="title" :anchor="anchor">{{ title }}</SectionTitle>
```

Comment:
We might need to do some additional investigation on this, but I suspect that we are now adding multiple elements with the same `id` value to the page since the `OnThisPageSection` is also using this `anchor` as an `id` for another HTML element, and there should only be one on the page since these are meant to be unique on a given page.

It might be as simple as removing it from the other component since it was previously used for the old "On This Page" affordance, which was removed with the introduction of the sidebar.

Reply:
#416 :) Not quite :D

Reply:
I know that component is coming back in a forthcoming PR, but we will need to ensure that we are only assigning a given unique ID to a single HTML element and not adding it in multiple places, regardless of whether or not there is an "On This Page" component being utilized.

Reply:
Right, I think you can check my explanation on how this component works and that might make things more clear. 
As long as we dont touch the OnThisPage registration logic, I am good 🤣

Reply:
Current logic should be enough to prevent duplication: https://github.com/apple/swift-docc-render/pull/408#discussion_r958671127